sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -328,6 +328,7 @@ public partial class ResponseClassifier
     {
         public ResponseClassifier() { }
         public virtual bool IsErrorResponse(Azure.Core.HttpMessage message) { throw null; }
+        public virtual bool IsRetriableException(Azure.Core.HttpMessage message, System.Exception exception) { throw null; }
         public virtual bool IsRetriableException(System.Exception exception) { throw null; }
         public virtual bool IsRetriableResponse(Azure.Core.HttpMessage message) { throw null; }
```

Comment:
I kind of like `IsRetriableResponse` more than `IsRetriableException` for the new overload because `HttpMessage + Exception` feels more like a possible `Response` than just an `Exception`.  I don't feel super strongly though.

Reply:
Hm, I thought about `IsRetriableResponse` as something that requires `message.Response` to be set.

Reply:
Maybe this one should just be `IsRetriable` since it's the union of the other two?  I'm fine with whatever you and Krzysztof prefer.

Reply:
I like IsRetriable